src/contentmanager.cpp

```diff
@@ -975,6 +998,12 @@ QStringList ContentManager::getBookIds()
     }
 }
 
+QString ContentManager::getRemoteLibraryUrl() const
+{
+  return makeHttpUrl(m_remoteLibraryManager.getCatalogHost(),
+                     m_remoteLibraryManager.getCatalogPort());
+}
+
```

Comment:
Choose a better place for this definition

Reply:
I think at the moment, makeHttpUrl should just be replaced by getRemoteLibraryUrl since it is only defined for the file scope and does the same thing anyway.

Reply:
Makes sense